package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mollie/api-client",
-  "version": "2.1.0",
+  "version": "2.1.0-alpha",
```

Comment:
alpha seems a versions that would be released as a test version prior to the v2.1.0?

Reply:
What version number would you suggest? alpha releases won't be installed by NPM automatically, so we can safely remove the tag from Git and this [specific version from NPM afterwards](https://docs.npmjs.com/cli/unpublish).

Reply:
I'm not sure why Travis doesn't run the tests though.